appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/SetMetricsConfigurationCommand.java

```diff
@@ -154,6 +180,46 @@ public void execute(AdminCommandContext context) {
         } catch (TransactionFailure ex) {
             actionReport.failure(LOGGER, "Failed to update Metrics configuration", ex);
         }
+
+        // If everything has passed, scrap the subaction reports as we don't want to print them out
+        if (!actionReport.hasFailures() || !actionReport.hasWarnings()) {
+            actionReport.getSubActionsReport().clear();
+        }
+    }
+
+    private boolean defaultMicroprofileUserExists(ActionReport subActionReport, Subject subject) {
```

Comment:
Both `defaultMicroprofileUserExists` and `createDefaultMicroprofileUser` did occur before. Can and should be shared for them?